optimum/tpu/generation/token_selector.py

```diff
@@ -118,6 +118,7 @@ def create(
             )
             generation_config.max_length = max_seq_length
 
+        generation_config._eos_token_tensor = None
```

Comment:
As said [here](https://github.com/huggingface/optimum-tpu/pull/87/files#r1738401721), could you replace this by this snippet?
```suggestion
    generation_config._eos_token_tensor = getattr(generation_config, "_eos_token_tensor", None)
```

Reply:
@tengomucho Above is review is from the old PR.  It is already updated. https://github.com/huggingface/optimum-tpu/pull/92/commits/10f857c4eadab57ad24a2d0a7c862c8aac5de4f5. I have replaced the snippet in [here](https://github.com/dstackai/optimum-tpu/blob/10f857c4eadab57ad24a2d0a7c862c8aac5de4f5/text-generation-inference/server/text_generation_server/generator.py#L168)